cmd/kubeadm/app/preflight/checks.go

```diff
@@ -912,25 +912,58 @@ func RunInitNodeChecks(execer utilsexec.Interface, cfg *kubeadmapi.InitConfigura
 		FileAvailableCheck{Path: kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.Etcd, manifestsDir)},
 		HTTPProxyCheck{Proto: "https", Host: cfg.LocalAPIEndpoint.AdvertiseAddress},
 	}
+
+	// File content check for IPV4 and IPV6 are needed if it is -
```

Comment:
```suggestion
	// File content checks for IPV4 and IPV6 are needed if it is -
```

Reply:
it reads as (File content check) (for IPV4 and IPV6 are needed) (if it is...)